src/Poseidon/CLI/OptparseApplicativeParsers.hs

```diff
@@ -762,3 +767,43 @@ parseMaybeArchiveName = OP.option (Just <$> OP.str) (
     OP.value Nothing <>
     OP.showDefault
     )
+
+parseJannocoalSourceSpec :: OP.Parser JannoSourceSpec
+parseJannocoalSourceSpec = parseJannocoalSingleSource <|> (JannoSourceBaseDirs <$> parseBasePaths)
+  where
+    parseJannocoalSingleSource = OP.option (JannoSourceSingle <$> OP.str) (
+        OP.long "sourceFile" <>
```

Comment:
The short form `-s` might be helpful here.

Reply:
Yes indeed. I've added `-s` and `-t` as suggested. And I've added `-o`. Thanks